target/linux/mvebu/patches-4.19/301-PCI-armada8k-add-support-for-gpio-controlled-reset-signal.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/drivers/pci/controller/dwc/pcie-armada8k.c b/drivers/pci/controller/dwc/pcie-armada8k.c
```

Comment:
This should be backport of upstream commit with proper message and don't forget to refresh patch with `make target/linux/{clean,refresh} V=s`.
On the other side, this patch got later reverted https://git.kernel.org/torvalds/c/f14bcc0add3abecceca1a3fe538c4ec9566893f3 for clashing with MACCHIATObin, is this patch needed for PCIe ports to function? And does this commit https://gitlab.denx.de/u-boot/u-boot/commit/d7f165cf676f4ae8aafd7adfc6dbe0f851026e46 make this patch obsolete? If yes, I would urge SolidRun to issue updated u-boot images on their site and mention in commit message that the bootloader needs re-flashing for PCIe ports to function.